docs/website/blog/2024-01-11-dlt-dbt-semantic-layer.md

```diff
@@ -0,0 +1,135 @@
+---
+slug: dlt-dbt-semantic-layer
+title: "dlt & dbt in Semantic Modelling"
+image: /img/blog-dbt_sem-data-people.png
+authors:
+  name: Hiba Jamal
+  title: Data Science intern at dlthub
+  url: https://github.com/hibajamal
+  image_url: https://avatars.githubusercontent.com/u/35984866?v=4
+tags: [semantic modelling]
+---
+
+<aside>
+💡 TLDR; Thanks to dbt’s new semantic layer feature we can now write “data pipeline” and “business user access” in the same sentence. In this article we explore how to create an end to end pipeline using the dlt library and dbt’s new feature.
```

Comment:
you highlight dbt and dlt as `dbt` and `dlt` hereinafter, so highlight it here as well